src/Core/Bases/A11yProperty.cs

```diff
@@ -47,6 +47,7 @@ public string TextValue
             }
         }
 
+        // TODO: This does not yet support different custom properties in concurrent scans
```

Comment:
nit: Is this a large enough item that we should file an issue? I'm concerned we'll forget about a TODO in the code if we don't act on it immedietly.